tools/user-scripts/src/index.ts

```diff
@@ -0,0 +1,9 @@
+import { Effect } from 'effect';
+import { UserRuntime, UserService } from './lib/user-scripts.js';
+
+export const deleteUser = (baseUrl: string, envId: string, userId: string, accessToken: string) =>
```

Comment:
top level api, we can adjust how these args are passed in as one object or whatever, just started it like this.